.gitignore

```diff
@@ -1,160 +1,31 @@
-# Created by .ignore support plugin (hsz.mobi)
```

Comment:
why are we updating this file. Looks like instead of adding your changes on top of current master, you just copy pasted entire folder of your changes.